src/components/HomeCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+interface HomeCardProps {
+  href: string
+  imgSrc: string
+  altText?: string
```

Comment:
Something is telling me we will never fill that one, let's keep it simple and remove it

Reply:
oki I removed it